src/org/python/util/jython.java

```diff
@@ -723,6 +723,11 @@ private static void addDefaultsFromEnvironment(Properties registry) {
         // Go interactive after script. (PYTHONINSPECT because Python scripts may set it.)
         addDefault(registry, "python.inspect", getenv("PYTHONINSPECT"));
 
+        // PYTHONDONTWRITEBYTECODE
```

Comment:
I put this in addDefaultsFromEnvironment as other env vars are read here but might be better elsewhere as it is not setting a registry entry.

Reply:
Let's do it here.